README.md

```diff
@@ -41,15 +41,15 @@ limitations under the License.
 <p>State-of-the-art Natural Language Processing for Jax, PyTorch and TensorFlow
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation, etc in 100+ languages. Its aim is to make cutting-edge NLP easier to use for everyone.
+🤗 Transformers provides thousands of pretrained models to perform NLP tasks such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
```

Comment:
```suggestion
🤗 Transformers provides thousands of pretrained models to perform tasks on texts such as classification, information extraction, question answering, summarization, translation, text generation and more in over 100 languages. Its aim is to make cutting-edge NLP easier to use for everyone.
```
I would like to avoid the repetition of NLP if possible (if you have a better suggestion here, please go ahead!)

Reply:
No, your one is good, and avoiding the repetition is probably a good idea!